netcdf/attributes.go

```diff
@@ -40,6 +45,22 @@ func (a Attr) Len() (n uint64, err error) {
 	return
 }
 
+// ValueString returns the value of the attribute value of type text.
+func (a Attr) ValueString() (val string, err error) {
```

Comment:
I tried using the code below, but unfortunately, it returns the content of the variable, and not the attribute itself. There is no 'ReadBytes' defined on the Attr type - so I keep the current implementation

	func (a Attr) ValueString() (val string, err error) {
		l, err := a.Len()
		if err != nil {
			return
		}

		data := make([]byte,l)
		err = a.v.ReadBytes(data)
		if err != nil {
			return
		}

		val = string(data)
		return
	}